service/orchestrator/metrics_test.go

```diff
@@ -834,16 +833,17 @@ func TestService_UpdateMetricConfiguration(t *testing.T) {
 		name    string
 		fields  fields
 		args    args
-		wantRes *assessment.MetricConfiguration
+		want    assert.ValueAssertionFunc
 		wantErr bool
```

Comment:
A general point: I would always check the explicit error message. The error can be thrown for all sorts of reasons.